src/highlight.js

```diff
@@ -36,7 +36,7 @@ https://highlightjs.org/
       aliases   = {};
 
   // Regular expressions used throughout the highlight.js library.
-  var noHighlightRe    = /^(no-?highlight|plain|text)$/i,
+  var noHighlightRe    = /^no-?highlight$/i,
       languagePrefixRe = /\blang(?:uage)?-([\w-]+)\b/i,
```

Comment:
Will removing plain and text affect existing users?

Reply:
Hmm 🤔 We might need to search source files in other projects via Github search.  I suspect there will be some, even if not document. We should strive to remain backwards compatible.

Reply:
I'm skeptical about the idea of searching... Too easy to miss something.
Let's choose another name. How about `output` or `plaintext`?

Reply:
`plaintext` works for me!

Reply:
Fine, I'll make changes.

Reply:
Appropriate it. I’ll give others a chance to weigh in, but would like to merge by end of the week. Thanks for your hard work here and for thinking this through.